src/video.cpp

```diff
@@ -489,19 +494,16 @@ void VideoSystem::ScrollCallback([[maybe_unused]] GLFWwindow *window, [[maybe_un
 void VideoSystem::MouseClickCallback([[maybe_unused]] GLFWwindow *window, int button, int action, [[maybe_unused]] int mods) {
 	assert(window == _video.window);
 
+	MouseButtons mouse;
```

Comment:
Name is far from optimal, a `mouse` is not just a button. It also has a position and sometimes 4 legs. That said, the alternative `button` is already used in the header so that is not an option either. (It's also a bit too general I think, but probably not enough to fix now.)

Likely a name that is close enough is and not used already is `mouse_button`. How does that sound to you?